### DISCUSSION TITLE
How to run multifit() in parallel?

### DISCUSSION THREAD

**Discussion by PZH525** [hyperspy/hyperspy#3160] [Q&A] [answered]
My current code is this, but it runs very slowly, how to make this run in parallel so that it can run faster on the CPU cluster?

m.multifit(optimizer="lm", bounded=True, iterpath='serpentine')

**Comment by ericpre** [comment, marked as answer]
You can try to use [SAMFire](https://hyperspy.org/hyperspy-doc/current/user_guide/model.html#smart-adaptive-multi-dimensional-fitting-samfire) but it is not very well documented
